tox.ini

```diff
@@ -45,20 +34,11 @@ commands=codecov -X gcov --file {toxworkdir}/coverage.xml
 
 [testenv:docs]
 extras =
-        docs
-        testing
+	docs
+	testing
 changedir = docs
 commands =
-        {envpython} -m sphinx \
-          -j auto \
```

Comment:
parallelism makes things faster

Reply:
Perhaps, though it may also be a premature optimization. If it's a generally-useful feature, then I'd rather see it implemented as the default in the builder and otherwise rely on defaults where the impact is small. I just ran the builds locally, and they complete in 11.2s for parallelized and 11.5s otherwise. So, I've already wasted more of my life evaluating the feature than enabling it will ever save :(. YAGNI.

Reply:
The docs here are quite small but for the skeleton, it'd be a good idea to have it on.